packages/electron41/makepkg-source-roller.py

```
@@ -0,0 +1,508 @@
+
+    def format_path(dep_name):
+        return dep_name if not use_relative_paths else f"{prefix}/{dep_name}"
+
+    real_deps = OrderedDict()
+    cipd_deps = {}
+    gcs_deps = {}
+    reverse_map = reverse_map or {}
+    vars = vars or {}
+    condition_vars = deps_module.vars | vars
+
+    def condition_is_met(condition):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `condition_is_met`

(ANN202)